app/graphql/mutations/invoices/update.rb

```diff
@@ -10,6 +10,7 @@ class Update < BaseMutation
 
       argument :id, ID, required: true
       argument :payment_status, Types::Invoices::PaymentStatusTypeEnum, required: false
+      argument :metadata, [Types::Invoices::Metadata::Input], required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `metadata` should appear before `payment_status`.